internal/agents/pi/adapter.go

```
@@ -486,3 +486,47 @@ func defaultStat(path string) statResult {
 	}
 	return statResult{isDir: info.IsDir()}
 }
+
+// gentlePiPackageSource returns the selected gentle-pi package source read-only.
+// It never calls mergePiSettingsFile and respects PI_CODING_AGENT_DIR via
+// CodeGraphPaths/AgentConfigPath. Project settings take precedence over user.
+func gentlePiPackageSource(cwd, agentDir string) string {
```

golangci-lint (2.12.2)

[error] func gentlePiPackageSource is unused

(unused)